test/NetworkTest.spec.ts

```diff
@@ -212,6 +208,44 @@ describe('Network Test', () => {
           .catch(validateError)
           .finally(done);
       }, 40000);
+
+      it('should return valid test results or an error when there is no camera', (done) => {
+        const realOTGetDevices = OT.getDevices;
+        OT.getDevices = (callbackFn) => {
```

Comment:
What's going on here?

Reply:
This is forcing the OT.getDevices() to list no video input devices (for one test).